app/views/content/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme.md

```diff
@@ -4,8 +4,9 @@ heading: "Teachers' pension scheme"
 description: |-
   The teachers' pension scheme is one of the most generous in the country. Learn more about the benefits of the teachers' pension scheme.
 image: false
-promo_content:
-    - content/train-to-be-a-teacher/promos/mailing-list-promo-salaries
+cta_mailinglist:
+  mailinglist:
+    text: Explore more benefits of teaching, and learn what it's really like to be in the classroom.
```

Comment:
Again, I know this is in prod but is there a better way to word this than Explore? That doesn't seem appropriate here?